okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -234,6 +260,11 @@ private boolean resolveRedirects(ProxyContext pc,
         -> a.getRoutingEntry().getPhaseLevel().compareTo(b.getRoutingEntry().getPhaseLevel());
     mods.sort(cmp);
 
+    if (skipAuth) {
+      pc.debug("Skipping auth, have cached token.");
+      mods.remove(0);
```

Comment:
Do we know that the mi with skipAuth is at position 0?

Reply:
I'm pretty sure it's always going to be the auth call, no?